examples/upgrade/blue-green-route53/eks-green/kubernetes/ecsdemo-crystal/limit-range.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: 'v1'
```

Comment:
Can we combine the blue and green limit-ranges into one common directory?

Reply:
yes